### DISCUSSION TITLE
Boundary assignment help!!!

### DISCUSSION THREAD

**Discussion by kizzle4shaw** [PMEAL/porespy#813] [Q&A] [answered]
hello. I have used the advanced snow pore network extraction method,  and i got the following results. Could someone please help me on how to assign the boundaries so I can perform a permeability calculation using openpnm because at the moment I. can't get it right on setting the inlet and outlet pores. Is there a step the I missed? 

<img width="488" alt="IMG_8319" src="https://user-images.githubusercontent.com/32852010/227862358-be2fc73b-40ff-4b9a-b20f-c75080255776.png">

**Comment by kizzle4shaw** [comment]
also the network has been extracted from 3D micro ct

**Comment by jgostick** [comment, marked as answer]
Try this example: https://porespy.org/modules/generated/generated/porespy.networks.snow2.html#

**Reply by kizzle4shaw** [reply]
Thank you. The function has provided min and max values of pore.  I believe these values can be used for labelling inlets and outlets for simulation.
